src/utils/filterEvents.js

```diff
@@ -0,0 +1,23 @@
+import { format, isAfter } from 'date-fns'
```

Comment:
👍 for creating the file, but I think it could be called just `events`, cause by this name I'd expect that it exported by default a fn called `filterEvents` when it fact it has named exports for filter both meetups or conferences. What's your opinion?

Reply:
Fixed :)

Reply:
Changed to getEvents and moved